Common/GameState.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace untitledplantgame.Common;
+
+/// <summary>
+/// Possible states of the game.
+/// <remarks>
+/// This class is a pseudo enum with properties. (unused)
+/// Class objects are nullable unlike enums. (alternatively create some "NONE" enum)
+/// </remarks>
+/// </summary>
+public sealed class GameState
+{
+	public static readonly GameState Gameplay = new GameState();
```

Comment:
I thought we wanted to name this "Freeroam"?

Reply:
Changed it to `FreeRoam`. Also moved "gamestate stuff" to its own directory

Reply:
LGTM